Include/cpython/code.h

```diff
@@ -38,8 +38,8 @@ typedef struct {
    Line instrumentation creates an array of
    these. One entry per code unit.*/
```

Comment:
Does `One entry per code unit` still hold for the new struct?

Reply:
Yes. The size of the entries varies, but there is the same number of them.

Reply:
Yes. The size of the entries varies, but there is the same number of them.

Reply:
But line instrumentation does not create an array of "these" right? It used to create multiple of these, now it creates one, but with a flexible length.

Reply:
Yes, I see what you mean. It is one struct, containing an array of entries. I'll update the comment.